crates/env/Cargo.toml

```diff
@@ -40,7 +40,7 @@ sha3 = { version = "0.10", optional = true }
 blake2 = { version = "0.10", optional = true }
 
 # ECDSA for the off-chain environment.
-secp256k1 = { version = "0.21.2", features = ["recovery"] }
+secp256k1 = { version = "0.21.2", features = ["recovery", "global-context"] }
```

Comment:
So from what I can tell `global-context` implies `std`, so we should probably have this as `optional = true`